vex-riscv/README.md

```diff
@@ -0,0 +1,39 @@
+<!--
+SPDX-FileCopyrightText: 2022 Google LLC
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# VexRiscv integration
+
+## Building
+
+For building the CPU, the following software needs to be installed and available
+in the `PATH`:
+
+- a recent JDK installation
+- SBT, the scala build tool
+- a recent C compiler
+- `verilator`
+- `make` for building the verilated library and FFI code
+
+## Notes for using the core:
+
+- VexRiscv has a "reset vector" for the instruction bus. This is the initial
+  PC that gets fetched. This address only gets presented to the IBUS after at
+  least one cycle of RST being asserted.
+
+- When using the verilated FFI version, there is a delay forced by the FFI layer.
```

Comment:
I don't see where this delay is forced, could you point it out for me?

Reply:
Okay that's bad wording on my part. The FFI doesn't force it itself, but instead it doesn't work if there's no introduced 1 cycle delay. This delay is the `emptyInput :- inputs` or the `register definedOutput output` in the CPU code.
I will change this sentence!

Reply:
great!